js-miniapp-sdk/src/modules/user-info.ts

```diff
@@ -49,7 +60,36 @@ export class UserInfo implements UserInfoProvider {
     return getBridge().getContacts();
   }
 
-  getAccessToken(audience: string, scopes: string[]): Promise<AccessTokenData> {
-    return getBridge().getAccessToken(audience, scopes);
+  getAccessToken(
+    audience: string,
+    scopes: string[]
+  ): Promise<AccessTokenData | MiniAppError> {
+    return getBridge()
+      .getAccessToken(audience, scopes)
+      .catch(error => {
+        try {
+          const miniAppError = parseMiniAppError(error);
```

Comment:
Shouldn't the types and parsing of the error be in the `js-miniapp-bridge` module? That way the parsing is bundled with the Android/iOS SDK and we don't need to worry about the Mini App having the an incompatible version of the JS SDK.

Reply:
Yeah, that makes sense now you mention it :)